modules/configuration/ajax/process.php

```diff
@@ -28,12 +28,26 @@
 $client->makeCommandLine();
 $client->initialize();
 
-$DB =& Database::singleton();
+$DB      =& Database::singleton();
+$pathIDs = getPathIDs();
 foreach ($_POST as $key => $value) {
     if (is_numeric($key)) { //update
         if ($value == "") {
             $DB->delete('Config', array('ID' => $key));
         } else {
+            // Validate that a path exists before entering into database.
+            if (in_array(intval($key), $pathIDs, true)) {
+                if (!file_exists($value)) {
```

Comment:
Should there be a check to ensure it is not *outside* of Apache reach and it is ok to use (i.e.: "/bin/...")

Reply:
@johnsaigle still missing in action

Reply:
@PapillonMcGill line 47.

Reply:
@johnsaigle right, seen it before but I must had some fog in the eyes this morning.